lndmanage/lib/fee_setting.py

```diff
@@ -277,6 +274,7 @@ def new_fee_policies(self, init=False) -> Tuple[dict, list]:
                 base_fee_msat_new = int(
                     min(self.params["max_base_fee"], base_fee_msat_new)
                 )
+            factor_base_fee = base_fee_msat_new / base_fee_msat
```

Comment:
`base_fee_msat` may be zero

Reply:
Wups, good catch!

Arguably it could also make sense to have the special case of factor = 1 when both are 0, but opted not to to keep the logic simpler.